README.md

```diff
@@ -32,6 +32,13 @@ We apply these files as such:
 $ cat ./images/{multus-daemonset.yml,flannel-daemonset.yml} | kubectl apply -f -
 ```
 
+Note: For crio runtime use multus-crio-daemonset.yml (crio uses /usr/libexec/cni as default path for plugin directory). Before deploying daemonsets,delete all default network plugin configuration files under /etc/cni/net.d
```

Comment:
Aneesh -- can we move this out of the `README.md` and into the `./docs/quickstart.md`? Just looking to keep README.md mimimal and maybe more options in the quickstart.